src/styles.css

```
@@ -288,14 +288,15 @@ body{background:var(--bg);color:var(--ink);font-family:'Plus Jakarta Sans',sans-
 .btn-clear-filters:hover{background:var(--orange);color:white;transform:translateY(-1px);box-shadow:0 4px 12px rgb(244 107 14 / 0.2)}
 
 /* ── MODAL ── */
-.modal-bg{position:fixed;inset:0;z-index:100;background:rgba(28,16,8,.5);backdrop-filter:blur(8px);display:none;align-items:center;justify-content:center;padding:20px}
+.modal-bg{position:fixed;inset:0;z-index:100;background:rgba(28,16,8,.5);backdrop-filter:blur(8px);display:none;align-items:center;justify-content:center;padding:20px;overflow:hidden;scrollbar-gutter:stable;}
```

Stylelint (17.11.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)